cmd/nerdctl/image_inspect.go

```diff
@@ -95,11 +102,32 @@ func ImageInspectAction(clicontext *cli.Context) error {
 		}
 	}
 
-	b, err := json.MarshalIndent(f.entries, "", "    ")
-	if err != nil {
-		return err
+	var tmpl *template.Template
+	switch format := clicontext.String("format"); format {
+	case "":
+		b, err := json.MarshalIndent(f.entries, "", "    ")
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(clicontext.App.Writer, string(b))
+	case "raw", "table":
+		return errors.New("unsupported format: \"raw\" and \"table\"")
+	default:
+		var err error
+		tmpl, err = templates.Parse(format)
+		if err != nil {
+			return err
+		}
+		if tmpl != nil {
+			for _, value := range f.entries {
+				b, err := json.Marshal(value)
+				if err != nil {
+					return err
+				}
+				fmt.Fprintf(clicontext.App.Writer, string(b)+"\n")
+			}
+		}
```

Comment:
Lots of the codes are same across `container_inspect.go` and `image_inspect.go`.
You can move the common parts to `fmtutil.go`

Reply:
ContainerInspectBashComplete and imageInspectBashComplete func are the same , are they necessary to put them into fmtutil.go?

Reply:
Can be another PR.